modules/ticket_selector/DisplayTicketSelector.php

```diff
@@ -218,7 +218,7 @@ public function display($event = null, $view_details = false)
             return $this->ticketSalesClosedMessage();
         }
         // is the event expired ?
-        $template_args['event_is_expired'] = $this->event->is_expired();
+        $template_args['event_is_expired'] = is_admin() ? false : $this->event->is_expired();
```

Comment:
this needs to be reversed so that the "falsey" result is the last part of the ternary, as is this is saying: "If true, then false"